src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx

```diff
@@ -13,6 +13,10 @@ redirects:
 
 You can install the [PHP agent](/docs/agents/php-agent/getting-started/introduction-new-relic-php) on a Docker container or other container to monitor one or more of your PHP applications. This is supported for containers that meet [PHP agent requirements](/docs/agents/php-agent/getting-started/php-agent-compatibility-requirements).
 
+<Callout variant="caution">
+ The daemon only transmits data to Newrelic periodically, once every minute. If you are standing up and tearing down containers often, ensure that you leave the daemon container running for enough time to transmit any remaining data.
```

Comment:
please use: `New Relic` :)

Wording suggestion: The PHP Agent's daemon transmits data to New Relic periodically during the minute-long [harvest cycle](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/getting-started/glossary#harvest-cycle). If you are standing up and tearing down containers often, ensure that you leave the daemon container running long enough to transmit any remaining data.



(btw it's described in this page: https://docs.newrelic.com/docs/apm/transactions/transaction-traces/introduction-transaction-traces/)